src/99-appendices/03-hed.md

```diff
@@ -26,54 +26,91 @@ onset	duration	HED
 ...
 ```
 
-The direct approach requires that each line in the events file must be
+The direct approach requires that each line in the events file be
 annotated. Since there are typically thousands of events in each experiment,
 this method of annotation is usually not convenient unless the annotations are
 automatically generated. In many experiments, the event instances fall into a
 much smaller number of categories, and often these categories are labeled with
 numerical codes or short names. It is therefore more convenient to associate
 the HED annotations with these categories and allow the analysis tools to make
 the association with individual event instances during analysis. To use this
-approach, your \_events.tsv file should associate a category (often called an
+approach, your `*_events.tsv` file should associate a category (often called an
 event code) with each event instance. Since BIDS allows an arbitrary number of
-columns to be included in an \_events.tsv file, you can make this association
+columns to be included in an `*_events.tsv` file, you can make this association
 by including columns representing various types of event categories in your
-\_events.tsv file.
+`*_events.tsv` file.
 
 Example:
 
 ```Text
 onset	duration	mycodes
-1.1	n/a	Fixation  
+1.1	n/a	Fixation
 1.3	n/a	Button
 1.8	n/a	Target
 ...
 
 ```
-
-If you provide an \_events.json file somewhere in your data hierarchy that has
-an HED mapping for `mycodes`, the HED tags associated with a given `mycodes`
-value can then be associated with the event instances in that category. You
-may provide a `HED` column and multiple category columns. The union of the
-relevant HED tags will then be associated with the event instance.
+ 
+The tags in the `HED` column of the `*_events.tsv` file are often specific to the individual event instances,
+while the common properties are represented by categorial values appearing in other columns. 
```

Comment:
```suggestion
while the common properties are represented by categorical values appearing in other columns.
```

Reply:
Corrected the spelling of categorical.  Not sure about the removing of the space?